sdk/search/Microsoft.Azure.Search/CHANGELOG.md

```diff
@@ -99,7 +99,7 @@ None
 
 ### Breaking Changes
 
-For more details, visit the [SDK migration guide](https://docs.microsoft.com/azure/search/search-dotnet-sdk-migration-version-9).
+For more details, visit the SDK migration guide.
```

Comment:
Are these links just gone? I'd think we would still want to link to _something_ if we can.

Reply:
the docs were archived but for some reason they don't show up in previous versions. They may eventually be fixed but there is not estimate as to when.

Reply:
The version is now version 11 and the link is here: https://docs.microsoft.com/en-us/azure/search/search-dotnet-sdk-migration-version-11. There are also hard links next to that page in the TOC for upgrading the rest api and mgmt sdk, but sadly the data plane sdk does not have a consistent link. @weshaggard @danieljurek does this seem like an oversight or something we can fix? I'm not the biggest fan of having to choose between links that will expire or referencing docs without a link and making the user find them.